### PR TITLE
Fix RenewInterval computation in ACME provider

### PR DESCRIPTION
**Pull request by smasset-orange**
### What does this PR do?

Fixes typos in ACME provider getCertificateRenewDurations and test comments:
- There are actually 365 days in a year
- Renew period for 1 year certificates is actually 4 months (the test comment says 2)

### Motivation

This aligns the code with the documentation

### More

- [x] Added/updated tests
- [ ] Added/updated documentation

### Additional Notes
First commit adds a test for 90d < certificateDuration  < 1y to highlight issue
Second commit actually fixes it to make the new and existing tests pass

### CODE REVIEW

**Comment by smasset-orange** [comment]
Waiting for workflow approval before pushing code fix

**Comment by smasset-orange** [comment]
Should I open separate PRs for v2.7 and master branches or will v2.6 be automatically merged into them ?

**Comment by ldez** [comment]
The PR will be backported, you don't need to open several PRs.

**Review by kevinpollet — APPROVED** [review]
Thanks 👍

**Review by rtribotte — APPROVED** [review]
Thanks 👍

**Review by tomMoulard — APPROVED** [review]
LGTM :ok_hand:
Thank you for this catch !

**Pull request merged**
